packages/machine/test/unit/cf-operations/fixture.ts

```diff
@@ -68,3 +58,70 @@ export const TEST_FREE_BALANCE = new cf.utils.FreeBalance(
   100,
   new cf.utils.Nonce(true, 0, 5)
 );
+
+export const TEST_FREE_BALANCE_APP_INTERFACE = cf.utils.FreeBalance.contractInterface(
+  TEST_NETWORK_CONTEXT
+);
+
+export const TEST_FREE_BALANCE_APP_INSTANCE = new cf.app.AppInstance(
+  TEST_NETWORK_CONTEXT,
+  TEST_MULTISIG_ADDRESS,
+  TEST_PARTICIPANTS,
+  TEST_FREE_BALANCE_APP_INTERFACE,
+  cf.utils.FreeBalance.terms(),
+  TEST_FREE_BALANCE.timeout,
+  TEST_FREE_BALANCE.uniqueId
+);
+
+export const TEST_APP_INSTANCE = new cf.app.AppInstance(
+  TEST_NETWORK_CONTEXT,
+  TEST_MULTISIG_ADDRESS,
+  TEST_PARTICIPANTS,
+  TEST_APP_INTERFACE,
+  TEST_TERMS,
+  TEST_TIMEOUT,
+  TEST_APP_UNIQUE_ID
+);
+
+export function constructContractCall(funcSig: string, ...args: any): string {
+  const [funcName] = funcSig.split("(");
+  return new ethers.utils.Interface([funcSig]).functions[funcName].encode(args);
+}
+
+export function constructMultiSend(subcalls: string[]): string {
+  return constructContractCall(
+    "multiSend(bytes)",
+    `0x${subcalls.map(call => call.substr(2)).join("")}`
+  );
+}
+
+export function constructMultiSendSubCall(
+  operation: "delegatecall" | "call",
+  to: string,
+  value: string | number,
+  funcSig: string,
+  args: any[]
+): string {
+  const data = constructContractCall(funcSig, ...args);
+  return ethers.utils.defaultAbiCoder.encode(
+    ["uint8", "address", "uint256", "bytes"],
+    [operation === "delegatecall" ? 1 : 0, to, value, data]
+  );
+}
+
+export function constructMultisigExecTransaction(
+  operation: "delegatecall" | "call",
+  to: string,
+  value: string | number,
```

Comment:
`value` isn't being used in this function.

Reply:
Oooops